components/engagement/EngSelectContacts.vue

```diff
@@ -41,65 +44,106 @@
         </div>
       </div>
     </form>
-    <div v-if="isSelected1" class="show-contact1">
-      <show-contacts
-        :contact-name="contactName"
-        :department="department"
-        :contact-email="contactEmail"
-        :last-eng-title="lastEngTitle"
-        :engagement-date="engagementDate"
-        :num-participants="numParticipants"
-        :is-selected="isSelected1"
-      />
-      <button @click="isSelected1=false">
-        {{ $t ('engSelect.remove') }}
-      </button>
+    <div v-if="isSelected === true">
+      <div>
+        <display-contact
+          :name="contactName"
+          :org-name="orgName"
+          :email="contactEmail"
+          :type="engagementType"
+          :date="engagementDate"
+          :participants="participants"
+        />
+      </div>
+      <div>
+        <button @click="isSelected=false">
+          {{ $t ('engSelect.remove') }}
+        </button>
+      </div>
     </div>
+    <!--
     <div v-if="isSelected2" class="show-contact2">
-      <show-contacts
-        :contact-name="contactName"
-        :department="department"
-        :contact-email="contactEmail"
-        :last-eng-title="lastEngTitle"
-        :engagement-date="engagementDate"
-        :num-participants="numParticipants"
-        :is-selected="isSelected2"
-      />
+      display contact info here
       <button @click="isSelected2=false">
         {{ $t ('engSelect.remove') }}
       </button>
     </div>
+    -->
   </div>
 </template>
 
 <script>
-import AppSelect from '../app/AppSelect'
-import showContacts from './EngShowContacts'
+import displayContact from '../app/AppDisplayContact'
```

Comment:
you should be using the component from task # 111

Reply:
This is fixed, I am using the component you created